e2e/tests/container_create.go

```diff
@@ -654,6 +656,157 @@ func ContainerCreate(opt *option.Option) {
 				}
 			}
 		})
+		It("should create container with specified blkio settings options", func() {
+			// define options
+			options.Cmd = []string{"sleep", "Infinity"}
+			options.HostConfig.BlkioWeight = 500 // valid values: 0-1000
+
+			// Create WeightDevice objects for input
+			weightDevices := []*blkiodev.WeightDevice{
+				{
+					Path:   "/dev/sda",
```

Comment:
This relies on the host to have the /dev/sda and /dev/sdb devices, which is dependent on the test instance. This may lead to inconsistent test results. We can instead create a test device similar to what was done in nerdctl https://github.com/containerd/nerdctl/blob/8d7539ab357d788af25722e54c6de79ad83f8fd1/cmd/nerdctl/container/container_run_cgroup_linux_test.go#L488

Reply:
Created a dummy device to run the e2e tests